spec/factories.rb

```diff
@@ -1,5 +1,20 @@
 FactoryGirl.define do
   factory :redirection do
+    sequence(:slug) { |n| "slug#{n}" }
+    sequence(:url) { |n| "http://example#{n}.com" }
+    next_id 0
+
+    after(:create) do |redirection|
+      if redirection.next_id != 0
```

Comment:
I still feel like we could use seeds, no? http://stackoverflow.com/a/19930700/638966

Then we can have seeds that we'll use for production anyway.